pamelalozano/src/main/webapp/script.js

```diff
@@ -27,14 +27,7 @@ function addRandomGreeting() {
   greetingContainer.innerText = greeting;
 }
 
-function projectToggle(id){
-    //the id for each button is proj[i], i being the index
-    //c-proj[i] is the id for the div with the description of each project
-
-    if($("#c-"+id)[0].style.display!="block"){
-        $("#c-"+id)[0].style.display="block"
-        location.href = "#c-"+id;
-    } else{
-          $("#c-"+id)[0].style.display="none"
-    };
-}
\ No newline at end of file
+document.getElementById("projects").addEventListener("click", function (event) {
+    event.target.closest('.project').classList.toggle("open");
```

Comment:
Since clicks on a ".project-desc" or the parent "projects" element will also be handled by this callback, you might want to also make sure the event.target is a ".navbar".

For example, if you click on "div#projects", it will not find a closest ".project" and return null. You'd get an error like "Cannot find property classList of null" or something.

Reply:
Oh you are right! It's done.